jenkins/seed.groovy

```diff
@@ -17,6 +17,8 @@ projects.each { project ->
                 repoOwner('cfpb')
                 repository(project.repo)
                 scanCredentialsId('github')
+                buildForkPRHead(buildForkPRHead = true)
+                buildForkPRMerge(buildForkPRMerge = false)
```

Comment:
I believe this should be `buildForkPRHead(true)` as opposed to `buildForkPRHead(buildForkPRHead = true)`